File: Week_01/movezeros/movezeros.java
class Solution {
    public void moveZeroes(int[] nums) {
        int temp = 0;
        for(int lastFindZero = 0, cur = 0; cur < nums.length; cur++) {
            if(nums[cur] != 0) {
                temp = nums[lastFindZero];
                nums[lastFindZero] = nums[cur];
                nums[cur] = temp;
                lastFindZero++;
            }
        }
    }
}

File: Week_01/movezeros/movezeros.go
func moveZeroes(nums []int)  {
    temp := 0
    lastFindZero := 0
	for cur := 0 ; cur < len(nums) ; cur++ {
            if nums[cur] != 0 {
                temp = nums[lastFindZero]
                nums[lastFindZero] = nums[cur]
                nums[cur] = temp
                lastFindZero++;
            }
	}
	return
}